src/pages/Transactions/IssueRequestsTable/IssueRequestModal/index.tsx

```diff
@@ -113,16 +113,21 @@ const IssueRequestModal = ({
               {/* TODO: could componentize */}
               <h4
                 className={clsx(
-                  'font-medium',
-                  { 'text-interlayDenim':
-                    process.env.REACT_APP_RELAY_CHAIN_NAME === POLKADOT },
-                  { 'dark:text-kintsugiSupernova': process.env.REACT_APP_RELAY_CHAIN_NAME === KUSAMA },
-                  'space-x-1'
+                  'space-x-1',
+                  'text-xl'
                 )}>
-                <span className='text-5xl'>
+                <span>
```

Comment:
note: the span elements are required to create space between the header elements (in preference to using `&nbsp;`).